backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/facet/impl/LongMultiValueFacetCounts.java

```diff
@@ -76,6 +77,11 @@ private void increment(long value) {
 		}
 	}
 
+	@Override
+	public FacetResult getAllChildren(String dim, String... path) throws IOException {
+		return getTopChildren( Integer.MAX_VALUE, dim, path );
+	}
```

Comment:
Maybe just throw an exception? I don't think we would use this method anyway. Same for other implementations.

Reply:
hah 😄 that was the second option I was considering 😄 will do